db/schema.rb

```diff
@@ -0,0 +1,89 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.1].define(version: 2024_02_10_110506) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "foods", force: :cascade do |t|
+    t.string "name"
+    t.string "measurement_unit"
+    t.decimal "price"
+    t.integer "quantity"
+    t.integer "user_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "inventories", force: :cascade do |t|
+    t.string "name"
+    t.string "description"
+    t.bigint "users_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["users_id"], name: "index_inventories_on_users_id"
+  end
```

Comment:
Please note that It is important to ensure that the schema of our app closely resembles the [Entity Relationship Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_erd.png) provided in the project core requirements. 

Based on the ERD diagram, each entity has specific parameters necessary. For instance, in the case of 'inventory', only 'Name' and 'User_id' are required. Could we omit the 'description' field you've included in inventories to align with the diagram? 🤗

Please make any other required adjustments to ensure that our app conforms to the ERD. 😊


![recipe_erd](https://github.com/faisalakb/food-recipe-group/assets/78251927/5fe3c6ed-6f87-46c7-9371-252ac80c337b)